apptrail/manifest.json

```diff
@@ -0,0 +1,79 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "302b6db7-d1d6-445c-ae20-00743775cb6b",
+  "app_id": "apptrail",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Security",
+    "Category::Log Collection",
+    "Category::Monitoring",
+    "Category::Alerting"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor, analyze, & alert on all your SaaS audit logs using Apptrail",
+    "title": "Apptrail",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Apptrail hosts an audit logs Portal on behalf of SaaS vendors, where customers can view, search, configure, and export their audit logs.",
+        "image_url": "images/1-at-portal.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "View a full audit event history from the Apptrail Portal, and search & filter by time and event properties.",
+        "image_url": "images/2-at-events-history.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Apptrail audit events contain detailed information on the who, what, where, when, and how of a recorded activity, alongside contextual information like IP address and custom event data.",
+        "image_url": "images/3-at-log-detail.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Create trails to continuously stream audit logs to dozens of destinations, like Datadog, in realtime for archival, monitoring, and analysis.",
+        "image_url": "images/4-at-create-trail-sel.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Use trail rules to filter and select a subset of events to deliver from a trail.",
+        "image_url": "images/5-at-trail-detail.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Apptrail audit logs are continously exported to Datadog Logs in realtime, where you can analyze, query, and monitor your audit logs.",
+        "image_url": "images/6-datadog-preview.png"
+      }
+    ]
+  },
+  "author": {
+    "name": "Apptrail",
+    "homepage": "https://apptrail.com",
+    "sales_email": "sales@apptrail.com",
+    "support_email": "support@apptrail.com"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "Apptrail",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "apptrail.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
```

Comment:
Hi @Samrose-Ahmed, since your integration is not sending in any metrics, you don't need the metadata.csv file or this integration object. When you remove this integration object and the metdata.csv file, you should be able to pass validation.